src/lib.rs

```diff
@@ -127,6 +127,20 @@ impl Alpm {
     })
   }
 
+  /// Compare two version strings and determine which one is newer.
+  ///
+  /// Returns a value comparable to the way strcmp works. Returns 1
```

Comment:
The line wrap was a bit confusing, but when it returns `std::cmp::Ordering` it will become shorter. This will be ok.